server/model/tournamentManager.ts

```diff
@@ -54,4 +54,49 @@ export class TournamentManager {
     }
     return players;
   }
+
+  public createNextBracket(): void {
+    // creating the next bracket in the tournament
+    const nextBracket = [];
+    if (this.tournamentBrackets[-1].length != 1) {
+      // if there is no winner yet
+      for (const currPlayer of this.tournamentBrackets[-1]) {
+        if (currPlayer.getWinstreak() != 0) {
+          nextBracket.push(currPlayer);
+        }
+      }
+    }
+    // adding new bracket into tournamentBrackets
+    this.tournamentBrackets.push(nextBracket);
```

Comment:
Kind of nitpicking, but this function seems slightly incorrect. As it will add a new bracket, even if no one is in it.